security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,39 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
+=== New features and enhancements
+
+* With this update, Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
+
+* With this update, obsolete rules being applied to `s390x` have been removed. (link:https://issues.redhat.com/browse/CMP-2471[*CMP-2471*])
+
+[id="compliance-operator-1-4-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, for RHCOS systems using RHEL9, application of the `upstream-ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` rule would fail. With this update, after auto remediation is applied, RHEL9-based RHCOS systems will show `PASS` upon the application of this rule. (link:https://issues.redhat.com/browse/OCPBUGS-13589[*OCPBUGS-13589*])
+
+* Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes are no longer accessible via SSH using the core user account. With this update, nodes remain accessible via SSH using sshkey. (link:https://issues.redhat.com/browse/OCPBUGS-1833[*OCPBUGS-1833*])
+
+* Previously, the `STIG` profile was missing rules from CaC that fulfill requirements on the published `STIG` for {product-title}. With this update, upon remediation, the cluster satisfies `STIG` requirements that can be remediated using Compliance Operator. (link:https://issues.redhat.com/browse/OCPBUGS-26193[*OCPBUGS-26193*])
+
+* Previously, creating a `ScanSettingBinding` with two profiles would override the prohibition against multiple product types. With this update, validation is now consistent regardless of the number of profiles listed in `ScanSettingBinding`. (link:https://issues.redhat.com/browse/OCPBUGS-26229[*OCPBUGS-26229*])
+
+* Previously, running rule `rhcos4-service-debug-shell-disabled` would show as `FAIL` even after auto-remediation is applied. With this update, rule `rhcos4-service-debug-shell-disabled` now shows `PASS` after auto-remediation is applied. (link:https://issues.redhat.com/browse/OCPBUGS-28242[*OCPBUGS-28242*])
+
+* Instructions for the use of rule `rhcos4-banner-etc-issue` have been enhanced to provide more helpful detail. (link:https://issues.redhat.com/browse/OCPBUGS-28797[*OCPBUGS-28797*])
+
+* Previously the `api_server_api_priority_flowschema_catch_all` rule would provide `FAIL` status on OpenShift 4.16 clusters. With this update, the `api_server_api_priority_flowschema_catch_all` rule provides `PASS` status on OpenShift 4.16 clusters. (link:https://issues.redhat.com/browse/OCPBUGS-28918[*OCPBUGS-28918*])
+
+* Previously, when a profile is removed from a completed scan shown in `ScanSettingBinding`, Compliance Operator does not remove the old scan. Afterward, when launching a new SSB using the deleted profile, Compliance Operator fails to update the result. With this update of Compliance Operator, the new SSB now shows the new compliance check result. (link:https://issues.redhat.com/browse/OCPBUGS-29272[*OCPBUGS-29272*])
```

Comment:
Similar comment. Instead of:

"Previously, when a profile is removed from a completed scan shown in `ScanSettingBinding`, Compliance Operator does not remove the old scan. Afterward, when launching a new SSB using the deleted profile, Compliance Operator fails to update the result. With this update of Compliance Operator, the new SSB now shows the new compliance check result."

Maybe use "Previously, when a profile is removed from a completed scan shown in `ScanSettingBinding`, the Compliance Operator did not remove the old scan. Afterward, when launching a new SSB using the deleted profile, the Compliance Operator failed to update the result. With this update of  the Compliance Operator, the new SSB shows the compliance check result."

Reply:
Agreed, done!